builder/rootfs/etc/confd/templates/check-repos

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+export ETCD=${ETCD:-$HOST:4001}
+
+cd $(dirname $0) # absolute path
+
+for repo in $(ls | grep .git)
+do
+    reponame="${repo%.*}"
+    if ! etcdctl -C $ETCD ls /deis/services/"$reponame" > /dev/null 2>&1
```

Comment:
As a result of the above error, `etcdctl` returns 2 in this case because it couldn't contact the `etcd` server. Really we should be testing only for 4, the RC that indicates the key or dir actually didn't exist.